docs/Using-Fleet/REST-API.md

```diff
@@ -5273,19 +5281,24 @@ Returns a list of all queries in the Fleet instance.
-| observer_can_run | bool    | body | Whether or not users with the `observer` role can run the query. In Fleet 4.0.0, 3 user roles were introduced (`admin`, `maintainer`, and `observer`). This field is only relevant for the `observer` role. The `observer_plus` role can run any query and is not limited by this flag (`observer_plus` role was added in Fleet 4.30.0). |
+| Name                        | Type    | In   | Description                                                                                                                                            |
+| --------------------------- | ------- | ---- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| id                          | integer | path | **Required.** The ID of the query.                                                                                                                     |
+| name                        | string  | body | The name of the query.                                                                                                                                 |
+| query                       | string  | body | The query in SQL syntax.                                                                                                                               |
+| description                 | string  | body | The query's description.                                                                                                                               |
+| observer_can_run            | bool    | body | Whether or not users with the `observer` role can run the query. In Fleet 4.0.0, 3 user roles were introduced (`admin`, `maintainer`, and `observer`). This field is only relevant for the `observer` role. The `observer_plus` role can run any query and is not limited by this flag (`observer_plus` role was added in Fleet 4.30.0). |
+| interval                   | integer | body | The amount of time, in seconds, the query waits before running. Can be set to `0` to never run. Default: 0.       |
+| platform                    | string  | body | The OS platforms where this query will run (other platforms ignored). Comma-separated string. If set to "", runs on all compatible platforms.                    |
+| min_osquery_version             | string  | body | The minimum required osqueryd version installed on a host. If omitted, all osqueryd versions are acceptable.                                                                          |
+| automations_enabled             | boolean | body | Whether to send data to the configured log destination according to the query's `interval`. |
+| logging             | string  | body | The type of log output for this query. Valid values: `"snapshot"`(default), `"differential", or "differential_ignore_removals"`.                        |
```

Comment:
@zhumo should this also accept an optional `team_id` param?

Reply:
I believe it should. @rachaelshaw, could you confirm?

Reply:
@jacobshandling good catch! Gave it a think, and am fairly sure we don't actually need it for anything right now:
+ In the UI, you always create a query in the context of a team (like with policies) and we won't have a way of transferring queries between teams at this time
+ In the config YAML, we'll know which query is being changed based on the combo of query name & team name, and applying the config isn't destructive (if a query or policy is missing from the config, we don't delete it) — so I think if a team changes in the query YAML, it would just be treated as a brand new query and would still exist on the previous team unless it's deleted via the API or UI. (Kind of a gap in a Fleet GitOps workflow, but at least it's hard to accidentally delete everything 😅)

When in doubt, I've been looking at what we have for policies, and there's no way to change the team in the "Edit policy" endpoint so... I think we should just leave it out.

Reply:
Ah, I see. This is on the update query endpoint, not the create query endpoint! Yeah, right now, the recommended way to change an existing query's team is to delete it and then add it to the other team... no transfers. Thanks all!